src/model.py

```diff
@@ -385,6 +393,28 @@ async def generate(self, request):
             ).as_numpy()[0]
             if isinstance(prompt, bytes):
                 prompt = prompt.decode("utf-8")
+
+            multi_modal_data_input_tensor = pb_utils.get_input_tensor_by_name(
+                request, "multi_modal_data"
+            )
+            if multi_modal_data_input_tensor:
+                multi_modal_data = multi_modal_data_input_tensor.as_numpy()[0].decode("utf-8")
+                multi_modal_data = json.loads(multi_modal_data)
+                if "image" in multi_modal_data:
+                    image_list = []
+                    for image_base64_string in multi_modal_data["image"]:
+                        if "base64," in image_base64_string:
+                            image_base64_string = image_base64_string.split("base64,")[-1]
+                        image_data = base64.b64decode(image_base64_string)
+                        image = Image.open(BytesIO(image_data)).convert("RGB")
```

Comment:
NOTE: May need to expose image formats other than RGB in the future, but seems like a sensible default / first support for now. We can probably defer exposing it until we have a use case requiring other formats.

ex: https://github.com/vllm-project/vllm/blob/1dd4cb2935fc3fff9c156b5772d18e0a0d1861f0/vllm/multimodal/utils.py#L33